scripts/import/import_variant_synonyms

```diff
@@ -202,6 +266,66 @@ sub get_source{
   return $source;
 }
 
+=head2 get_all_sources
+
+get sources object 
+
+=cut
+
+sub get_all_sources{
+
+  my $dest_db = shift;
+  my $prod_dbc = shift; 
+
+  # my %source_names; # hash with source id and corresponding source name 
+  my %source_list; 
+
+  my $source_adaptor = $dest_db->get_SourceAdaptor('human', 'variation'); 
+
+  my $source_output = query_synonyms_data($prod_dbc, '*', 'source');  
```

Comment:
Could you specify the columns here to ensure they are returned in the expected order? We have had problems when patches changed the default order returned in the past.